utils/tls.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"io/ioutil"
+
+	"github.com/golang/glog"
+	"google.golang.org/grpc/credentials"
+)
+
+func GetMutualTLSCredentials(certificateFile string, keyFile string,
```

Comment:
I am not quite follow what this function does, in what case we should read cert from file? And how about renaming this func to GetTLSCredentials? It would be more generic. See the last line of this func, credentials.NewTLS